src/picongpu/include/simulation_defines/param/ionizationEnergies.param

```diff
@@ -20,38 +20,64 @@
 
 /** \file
  * This file contains the ionization energies and conversion to corresponding
- * electric field strengths for different species. */
+ * electric field strengths for different species.
+ */
 
 #pragma once
 
 /** \todo Make compatible for multiple atom species
  * - Probably do some calculation from energy to required field strength (or do that model dependent)
  * - Introduce some clever way of storage, like matrices for different charge states of the same species
- * - future: even different states of excitation should be possible */
+ * - future: even different states of excitation should be possible
+ */
 
 namespace picongpu
 {
-namespace SI
+namespace AU
 {
 
-    /* ionization energy for ground state hydrogen: 13.6 eV --> converted to Joule */
-    /* Joule = kg * m^2 / s^2 */
-    PMACC_CONST_VECTOR(float_X,1,IONIZATION_ENERGY,
-        2.179e-18
-    );
-    /* ionization field strength (E-field) for hydrogen: in Volt / meter
+    /** Ionization potentials
+     *
+     * Please follow these rules for defining ionization energies of atomic species,
+     * unless your chosen ionization model requires a different unit system than AU:
+     *     - enter ionization energies in SI units
```

Comment:
Hey @ax3l ... since I'm compile-time-converting the input to atomic units anyway, we can also let the user enter the ionization energies for their species in eV, can't we?   
It doesn't matter so much, of course but they are easier to find online and one doesn't have to calculate them in Joules.